github/activity_events_test.go

```diff
@@ -284,8 +284,12 @@ func TestActivityService_EventPayload_typed(t *testing.T) {
 	}
 
 	want := &PushEvent{PushID: Int(1)}
-	if !reflect.DeepEqual(event.Payload(), want) {
-		t.Errorf("Event Payload returned %+v, want %+v", event.Payload(), want)
+	got, err := event.ParsePayload()
+	if err != nil {
+		t.Fatalf("Payload = %v, want nil", err)
```

Comment:
This should now say "ParsePayload", and perhaps be more clear that it was the error returned. How about:

```Go
t.Fatalf("ParsePayload returned unexpected error: %v", err)
```

Consistent with https://github.com/google/go-github/blob/cfbfe91dfc6f331a809a2c942f64b6bad65d9c91/github/github_test.go#L631.